### ISSUE TITLE
Scrolling from the top and keeping the same perceptual scroll position

### ISSUE THREAD

**Issue by reivilibre** [oumoussa98/vue3-infinite-loading#19]
Hi and thanks for this library!

I'm wondering how you can use this library to scroll from the top whilst not loading all the historical data because the scroll position stays at the top even after new items are added.

I think this means that the scroll position needs to be recalculated when we add the new elements, but I don't know how to do this, or at least not how to do this with Vue.

I'm new to Vue and would really appreciate any clues — thank you :-)

**Comment by oumoussa98** [comment]
hello @reivilibre, can you please elaborate more on the issue.
If you can provide a simple demo would help a lot, thanks.

**Comment by reivilibre** [comment]
Hi,
I made a simpler example and noticed it worked.

I then realised that the problem was that I didn't set `target` properly so this library wasn't scrolling the right element.
Fixing the `target` attribute seems to have mostly fixed the problem. I think the rest of my problem is because the elements change size shortly after being created (they load some content from a server), so I need to think about pre-loading that and making sure they're ready to draw immediately.

Sorry for bothering you!
